src/encoded/static/components/browse/components/FileEntryBlock.js

```diff
@@ -368,13 +369,14 @@ export class FilePairBlock extends React.PureComponent {
     }
 
     render(){
-        const { files, columnHeaders, colWidthStyles, isSingleItem, excludeChildrenCheckboxes, hideNameOnHover, stackDepth } = this.props;
+        const { files, columnHeaders, colWidthStyles, isSingleItem, excludeChildrenCheckboxes, hideNameOnHover, stackDepth, columnClass } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['columnClass' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4901542)